ui/lib/kv/addon/components/page/secret/metadata/version-diff.hbs

```diff
@@ -0,0 +1,31 @@
+<KvPageHeader @breadcrumbs={{@breadcrumbs}} @pageTitle="Version Diff">
+  <:toolbarFilters>
+    <KvVersionDropdown
```

Comment:
I always was confused by the ordering here, and expected to read left to right as old to new. That might not be everyone's experience, but in any case I think we should add labels? (from/to)

Reply:
Yeah - this is what the original original design was but I'd agree that switching _does_ make a lot more sense. Also I think labels would help. - will add!